qiskit/pulse/library/symbolic_pulses.py

```diff
@@ -723,8 +722,10 @@ class Gaussian(metaclass=_PulseType):
 
     .. math::
 
+        \begin{align*}
```

Comment:
I don't know precisely how the Sphinx is converted, but I'd expect that the `.. math::` block introduces an outer `\begin{equation}` already. If that's the case, would `\begin{aligned}` a) be more appropriate and b) work with whatever backend you have?

Reply:
Yes, you're right. I've changed it from `align*` to `aligned` everywhere.